src/app/app.component.html

```diff
@@ -20,6 +20,15 @@
             </ion-item>
           </ion-menu-toggle>
         </ion-list>
+        <ion-menu-toggle auto-hide="false">
+          <ion-item button [routerDirection]="'root'" (click)="rollout()">
+            <ion-icon slot="start" name="log-out"></ion-icon>
+            <ion-label>
+              Roll Out
```

Comment:
Logout is a little different since there's no page associated with it, really. So it has an on-click event listener which will signout the user and navigate to the login page.